ext/ldap/ldap.stub.php

```diff
@@ -292,39 +292,39 @@ function ldap_8859_to_t61(string $value): string|false {}
 
 #ifdef HAVE_LDAP_EXTENDED_OPERATION_S
 /**
- * @param resource $link
+ * @param resource $ldap
  * @param string $retdata
  * @param string $retoid
  * @return resource|bool
  */
-function ldap_exop($link, string $reqoid, ?string $reqdata = null, ?array $servercontrols = [], &$retdata = null, &$retoid = null) {}
+function ldap_exop($ldap, string $reqoid, ?string $reqdata = null, ?array $controls = [], &$retdata = null, &$retoid = null) {}
```

Comment:
Would using a `$request_` prefix work well? And `$response_data` or something similar?